machine-learning/solutions/seismic_salt_detection/trainer/model.py

```diff
@@ -73,51 +73,40 @@ def create_regressor(config, parameters):
     Returns:
```

Comment:
Putting a TODO into this docstring sounds nice.  I'm also not sure, reading the docstring, what is/shouldn't be necessary and what the more-desirable code would look like.

Reply:
Also, since CL/187793590 has been committed, the text of the docstring could change a little.

Reply:
(this comment is meant to be attached to "forward_key_to_export")

Reply:
is CL/187793590 in production/public? if so for which version of TF?

Brian made me get rid of all TODOS last time :) (I had one there)

Reply:
Bleh -- sorry for the conflicting guidance.  Do you remember Brian's rationale for removing TODOs?

That CL was submitted back in March.  Seems awfully likely it's live, but I don't know of any way of tying a file's Piper version to the released TF version, but I could go through GitHub to find this change.

I think 2 things need to be done, here.  The first is to describe why this is a workaround and what should be done when that CL hits live (This comment should not have been submitted as written back when Lak wrote it).  The second is to do check if it's hit live and, if it has, either do that thing or note that we're no longer waiting on the CL but just the work to do what's described in thing 1.

Reply:
Brian said no complete solution should be published with a TODO as it means the solution wasn't ready... (or something along those lines)

I'm asking Lak, if it is live we can delete the whole function :)

I'll let you know